pages/common/mogrify.md

```diff
@@ -0,0 +1,23 @@
+# Mogrify
+
+> Tool that allows making operations on multiple images ie. resizing, cropping, flipping, adding effects. Replaces the original file.
```

Comment:
Would `viz.` be a better replacement for `ie.` ?

Reply:
Coming late to the discussion, but IMO we should avoid abbreviations if possible (and in any case, the spelling is `i.e.`, not `ie.`. I'll make a PR suggesting a change to `such as`, which IMO is the most beginner-friendly option :)